### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by CindyDorantes**
## Project requirements   :bookmark_tabs: 

- Take a look at the provided database diagram for a clinic.
- Create a file named `schema_based_on_diagram.sql` where you implement the database from the diagram.
- Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Remember to add the FK indexes.
- Add indexes for Foreign Keys

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+);
+
+CREATE TABLE treatments (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    type VARCHAR,
+    name VARCHAR,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE treatments_history (
```
Comment (omarsalem7) [inline schema_based_on_diagram.sql:44]: - [x] Consider adding indexes for all foreign key
> for example `CREATE INDEX ON medical_histories (patient_id);`

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi Team 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (CindyDorantes) [reply schema_based_on_diagram.sql:44]: Thanks! Done 😄 :

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi team, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**